spk/tt-rss/src/conf/resource

```diff
@@ -10,17 +10,5 @@
 			"relpath": "etc/php/conf.d/com.synocommunity.tt-rss.ini",
 			"destpath": "/usr/local/etc/php70/conf.d"
 		}]
-    },
-    "php72": {
```

Comment:
Those php workers actually don't exist. Therefore the ttrss configuration need to be manually configured at the level of the PHP profile :(